packages/jest-runtime/src/writeFileAtomic.js

```diff
@@ -0,0 +1,219 @@
+/* eslint-disable */
+
+// THIS FILE IS COPIED FROM https://github.com/npm/write-file-atomic/blob/1597785cb9e1152056d905a357b3891b3295506d/index.js
```

Comment:
somebody needs to tell me how to properly attribute this.

License: https://github.com/npm/write-file-atomic/blob/1597785cb9e1152056d905a357b3891b3295506d/LICENSE

Reply:
Not a lawyer, but as with MIT, we'll definitely need to copy over the license text

Reply:
done